scripts/deployed_general_agent_viewer.py

```diff
@@ -0,0 +1,119 @@
+"""
+PYTHONPATH=. streamlit run deployed_agent_viewer.py
+
+Tip: if you specify PYTHONPATH=., streamlit will watch for the changes in all files, instead of just this one.
+"""
+
+
+# Imports using asyncio (in this case mech_client) cause issues with Streamlit
+from prediction_market_agent.tools.streamlit_utils import (  # isort:skip
+    display_chat_history,
+    streamlit_asyncio_event_loop_hack,
+)
+
+streamlit_asyncio_event_loop_hack()
+
+# Fix "Your system has an unsupported version of sqlite3. Chroma requires sqlite3 >= 3.35.0" error
+from prediction_market_agent.utils import patch_sqlite3  # isort:skip
+
+patch_sqlite3()
+
+
+from datetime import datetime
+
+import streamlit as st
+from prediction_market_agent_tooling.markets.markets import MarketType
+
+from prediction_market_agent.agents.microchain_agent.memory import DatedChatHistory
+from prediction_market_agent.agents.microchain_agent.utils import get_total_asset_value
+from prediction_market_agent.agents.utils import AgentIdentifier
+from prediction_market_agent.db.long_term_memory_table_handler import (
+    LongTermMemoryTableHandler,
+)
+from prediction_market_agent.tools.streamlit_utils import check_required_api_keys
+from prediction_market_agent.utils import APIKeys
+
+
+class DeployedGeneralAgentKeys(APIKeys):
+    START_TIME: str
+    STARTING_BALANCE: float
+
+    @property
+    def start_time(self) -> datetime:
+        return datetime.fromisoformat(self.START_TIME)
+
+    @property
+    def starting_balance(self) -> float:
+        return self.STARTING_BALANCE
```

Comment:
if `self.STARTING_BALANCE` is already non-optional, why not just use it?

Reply:
just aliasing to follow the 'upper-case for env vars, lower case for properties' convention